packages/components/badge/src/EntityStatusBadge/EntityStatusBadge.tsx

```diff
@@ -14,19 +14,18 @@ const statusMap: { [key in EntityStatus]: BadgeVariant } = {
 };
 
 export interface EntityStatusBadgeProps
-  extends Omit<BadgeProps, 'variant' | 'children'> {
-  size?: BadgeSize;
+  extends Omit<BadgeProps, 'variant' | 'children' | 'endIcon' | 'startIcon'> {
```

Comment:
Any reasons why are we omitting the icons here? Are we certain we don't want icons with EntityStatusBadge?

Reply:
I didn't actually touch this part, I'm trying to find a bit of context around this, to be sure 🤔

Reply:
That was probably me; I asked @denkristoffer on the contentful slack channel, and he said that it probably didn't need to be added.
![image](https://user-images.githubusercontent.com/36935801/209064713-0c0bd892-5a16-414e-af25-cff46c9611a3.png)

I believe the reason is that the status badges already have a predetermined design which doesn't include using icons. See the [storybook](https://f36-storybook.contentful.com/?path=/story/components-entitystatusbadge--overview) for and example of this. 

Of course, the icons can always be added in the future.